frontend/fingertips-frontend/package.json

```diff
@@ -52,18 +52,13 @@
     "typescript": "^5.7.2",
     "zod": "^3.23.8"
   },
+  "engines": {
+    "npm": ">=10.9.0"
+  },
   "overrides": {
-    "govuk-react": {
-      "react": "$react",
-      "react-dom": "$react-dom"
-    },
-    "styled-components": {
-      "react": "$react",
-      "react-dom": "$react-dom"
-    },
-    "@testing-library/react": {
-      "react": "$react",
-      "react-dom": "$react-dom"
-    }
+    "react": "$react",
+    "react-dom": "$react-dom",
+    "ajv": "^8.17.1",
+    "whatwg-url": "^14.0.0"
```

Comment:
See https://dev.to/asim_khan_cbe65e41bcbbc65/solving-the-punycode-module-is-deprecated-issue-in-nodejs-2e59
this was to resolve the deprecated punycode warning when running test